Makefile

```diff
@@ -250,17 +219,21 @@ dos-api-gateway-clean: ### Clean event processor lambda docker image directory
 # Deployments
 
 sls-only-deploy: # Deploys all lambdas - mandatory: PROFILE, VERSION=[commit hash-timestamp]
-	eval "$$(make populate-deployment-variables)"
+	eval "$$(make -s populate-deployment-variables)"
 	make serverless-deploy
 
+quick-build-and-deploy:
```

Comment:
Can we add a description to this make target. Would be good to indicate why it's quick e.g. `Builds application but only deploys on sls`